astropy/cosmology/parameter.py

```diff
@@ -0,0 +1,265 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import astropy.units as u
+
+__all__ = ["Parameter"]
+
+
+class Parameter:
+    r"""Cosmological parameter (descriptor).
+
+    Should only be used with a :class:`~astropy.cosmology.Cosmology` subclass.
+
+    Parameters
+    ----------
+    fget : callable[[type], Any] or None, optional
+        Function to get the value from instances of the cosmology class.
+        If None (default) returns the corresponding private attribute.
+        Often not set here, but as a decorator with ``getter``.
+    fvalidate : callable[[object, object, Any], Any] or {'default', 'float'}, optional
+        Function to validate the Parameter value from instances of the
+        cosmology class. If "default", uses default validator to assign units
+        (with equivalencies), if Parameter has units. If "float" will first do
+        units, then take the float value.
+        Often not set here, but through a decorator with ``validator``.
+    doc : str or None, optional
+        Parameter description. If 'doc' is None and 'fget' is not, then 'doc'
+        is taken from ``fget.__doc__``.
+    unit : unit-like or None (optional, keyword-only)
+        The `~astropy.units.Unit` for the Parameter. If None (default) no
+        unit as assumed.
+    equivalencies : `~astropy.units.Equivalency` or sequence thereof
+        Unit equivalencies for this Parameter.
+    fmt : str (optional, keyword-only)
+        `format` specification, used when making string representation
+        of the containing Cosmology.
+        See https://docs.python.org/3/library/string.html#formatspec
+    derived : bool (optional, keyword-only)
+        Whether the Parameter is 'derived', default `False`.
+        Derived parameters behave similarly to normal parameters, but are not
+        sorted by the |Cosmology| signature (probably not there) and are not
+        included in all methods. For reference, see ``Ode0`` in
+        ``FlatFLRWMixin``, which removes :math:`\Omega_{de,0}`` as an
+        independent parameter (:math:`\Omega_{de,0} \equiv 1 - \Omega_{tot}`).
+
+    Examples
+    --------
+    For worked examples see :class:`~astropy.cosmology.FLRW`.
+    """
+
+    _registry_validators = {}
+
+    def __init__(self, fget=None, fvalidate="default", doc=None, *,
```

Comment:
I don't think `fget` is used at all anymore (and if it is, would not do anything), so remove!?

Reply:
Indeed!

Reply:
That was a big oversight. Thanks for catching!

Reply:
Done!